mobile_back_tflite/cpp/backend_tflite/BUILD

```diff
@@ -62,16 +62,22 @@ cc_library(
 cc_library(
     name = "neuron_delegate",
     srcs = select({
-        "@org_tensorflow//tensorflow:android_arm64": ["neuron/libs/arm64-v8a/libtensorflowlite_neuron_jni.so"],
+        "@org_tensorflow//tensorflow:android": [
+            "@neuron_delegate//neuron/java:libtensorflowlite_neuron_jni.so",
```

Comment:
Please, also update the following lines:
https://github.com/mlcommons/mobile_app_open/blob/84d1e61e83ce5277118ef2e2211ed37b8b0449a4/mobile_back_tflite/tflite_backend.mk#L29-L31
Currently the command `make flutter/android WITH_MEDIATEK=1` is failing on this branch because it can't find `libtensorflowlite_neuron_jni.so` to copy it.